pinot-common/src/main/antlr4/org/apache/pinot/pql/parsers/PQL2.g4

```diff
@@ -78,6 +78,7 @@ predicate:
   | isClause                              # IsPredicate
   | regexpLikeClause                      # RegexpLikePredicate
   | textMatchClause                       # TextMatchPredicate
+  | likeClause                            # TextMatchPredicate
```

Comment:
This parser is for PQL which is already deprecated. Let's not add new syntax to PQL. The calcite SQL parser already supports LIKE syntax, and we just need to wire the query engine

Reply:
+1

Reply:
SQL LIKE and TEXT_MATCH are. I am not sure about REGEXP